opentasks/src/main/res/values-fr/strings.xml

```diff
@@ -248,6 +251,47 @@
     <!-- Incoming share -->
     <string name="share_as_task">Ajouter une tâche</string>
 
+    <!-- Settings -->
+    <!-- Notification enable/disable switch label in settings -->
+    <string name="opentasks_settings_notification_enable">Notifications</string>
+    <!-- Section title in settings for the notification signals -->
+    <string name="opentasks_settings_notification_signals_section_title">Signaux de notification</string>
+    <!-- Notification Sound enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_sound">Son</string>
+    <!-- Notification Vibration enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_vibrate">Vibrer</string>
+    <!-- Notification "Pulse light" enabling/disabling entry label in settings -->
+    <string name="opentasks_settings_notification_lights">Clignotement LED</string>
+
+    <!-- Outgoing share -->
+    <string name="opentasks_share_footer">Partagé via OpenTasks</string>
+
+    <!-- GET_ACCOUNTS permission (in CONTACTS permission group) requesting dialog: -->
+    <string name="opentasks_permission_request_dialog_getaccounts_title">Permission requise</string>
+    <string name="opentasks_permission_request_dialog_getaccounts_message">Afin de lire la liste des comptes qui supportent la synchronisation des tâches sur votre appareil, %s a besoin de la permission GET_ACCOUNTS.<br/><br/>     Veuillez noter qu\'en raison d\'un choix malheureux de la part de Google, la demande de permission sera labelisée &quot;accéder à vos contacts&quot; à la place de &quot;lire vos comptes&quot;. Voir <a href="https://issuetracker.google.com/issues/37065963">Issue 37065963</a> pour toute information complémentaire.</string>
+    <string name="opentasks_permission_request_dialog_getaccounts_button_continue">Poursuivre</string>
+    <string name="opentasks_permission_request_dialog_getaccounts_button_settings">Ouvrir Paramètres</string>
+
+    <!-- Format indicating a relative expression and time. Example: "4 hours ago, 11:00 am" -->
+    <string name="opentasks_relative_time">%1$s, %2$s</string>
+    <!-- Format indicating a date and time. Example: "23 Aug, 11:00 am" -->
     <string name="opentasks_date_time">%1$s à %2$s</string>
 
 </resources>
+
+<!-- File "opentasks-provider/src/main/res/values/strings.xml" -->
```

Comment:
This part should go into a separate file `opentasks-provider/src/main/res/values-fr/strings.xml`